sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/implementation/util/Utility.java

```diff
@@ -166,4 +174,36 @@ public static String parseResultId(String operationLocation) {
     public static String generateRandomModelID() {
         return UUID.randomUUID().toString();
     }
+    public static Context enableSyncRestProxy(Context context) {
```

Comment:
should we add a null check same as getTracingContext() method below?

Reply:
we always do enableSYncRestProcy(getTracingContext(context)) so the null check is already taking place. Since every call needs tracing context the null check is added to that method and not enable sync.

Reply:
If so, should we merge the two methods together?

Reply:
getTracingContext is used individually too in Async clients.